data/json/npcs/holdouts/NC_GLOOSCAP.json

```diff
@@ -0,0 +1,389 @@
+[
+  {
+    "type": "npc",
+    "id": "ABENAKI_Glooscap",
+    "//": "Hunter with indigenous ancestry and big ideas.",
+    "name_unique": "Glooscap",
+    "class": "NC_HUNTER",
+    "attitude": 0,
+    "mission": 8,
+    "chat": "TALK_GLOOSCAP",
+    "faction": "no_faction"
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GLOOSCAP",
+    "dynamic_line": {
+      "u_has_var": "u_met_glooscap",
+      "type": "general",
+      "context": "meeting",
+      "value": "yes",
+      "yes": [
+        "Good to see you again, how can I help you?",
+        "They say I turned a man into a cedar tree once.  That's a terrible thing to say.",
+        "You like maple syrup?  That was me.  Not the syrup, that was the trees, but I figured out how to make it much harder to concentrate it, used to be way too easy.  Pretty nice right?",
+        "Did I tell you the one about the beavers?  Used to be way too big.  Like giant beavers, it was a mess.  I fixed that.  Of course there seem to be a bunch of new giant things out here now.  I'll have to think about that.",
```

Comment:
Should there be a comma between "Like" and "giant"?

Reply:
Not in this case, but that would be appropriate for people who use "like" as a placeholder while they are thinking. I use "um" instead personally.